### PR TITLE
 Expose more options for TF-TRT optimization

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

File: src/core/model_config.proto [hunk 1]
```diff
@@ -496,6 +496,18 @@ message ModelOptimizationPolicy
     //@@         The value can be one of "FP32", "FP16" and "INT8".
     //@@         Default value is "FP32".
     //@@
+    //@@         "use_calibration" Whether to perform INT8 quantization with
+    //@@         TensorRT calibration. If set to False, quantization nodes are
+    //@@         expected for every tensor in the graph. Default value is true.
+    //@@         This argument is ignored if 'precision_mode' is not 'INT8'.
+    //@@
+    //@@         "is_dynamic_op" Whether at least one input has dynamic shape.
```
Comment (deadeyegoodwin) [inline src/core/model_config.proto]: If we infer this value from the model shape why does it need to be specified explicitly? Perhaps the description is incorrect?

Comment (GuanLuo) [reply src/core/model_config.proto]: The edge case that I am considering is that the model input is dynamic shape but the user specify it as fixed-size tensor in model config. In that case, `is_dynamic_op` should be true but inferring from model config says false.

Comment (deadeyegoodwin) [reply src/core/model_config.proto]: OK to keep it then

File: src/backends/tensorflow/base_backend.cc [hunk 2]
```diff
@@ -237,48 +239,47 @@ BaseBackend::CreateExecutionContext(
...
+                &tftrt_config.is_dynamic_op_));
           } else {
             return Status(
                 RequestStatusCode::INVALID_ARG,
                 "unknown parameter '" + parameter.first +
                     "' is provided for TensorRT Execution Accelerator");
           }
         }
+        // If needed, set 'use_calibration' at hindsight
+        if (!is_calibration_specified) {
```
Comment (deadeyegoodwin) [inline src/backends/tensorflow/base_backend.cc]: Remove this and other calibration-related settings

**Pull request merged**